### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by PascalPflaum**
Repaired the gulp target by replacing eshint with eslint.

I tried to find a good ruleset for the current code. I took [StandardJS](https://standardjs.com) with some minor adjustments to those rules. For many rules there were different styles in the repo (like comma-dangle), so I took the setting value based on my personal opinion in those cases.

### CODE REVIEW

**Comment by PascalPflaum** [comment]
Please notice that this branch includes #147.

File: src/index.js [hunk 1]
```diff
@@ -7,13 +7,14 @@ import { EventEmitter } from 'events';
...
 import JSONStorage from './storages/JSONStorage';
+import MongoDBStorage from './storages/MongoDBStorage';
 import SequelizeStorage from './storages/SequelizeStorage';
 
 /**
  * @class Umzug
  * @extends EventEmitter
  */
-module.exports = class Umzug extends EventEmitter {
+export default class Umzug extends EventEmitter {
```
Comment (jukkah) [inline src/index.js]: Double check that this is actually working? I don't want to broke the package again.

Comment (PascalPflaum) [reply src/index.js]: You are right. So used to import/export that I totally forgot about classic require.
I will check it and will try to add a way to automatic test this. First idea I came up with is splitting up tests in unit tests (running against src) and integration tests (running against lib).

**Comment by jukkah** [comment]
Ref #146

**Pull request merged**